toolchain/check/eval.cpp

```diff
@@ -497,6 +510,30 @@ static auto PerformBuiltinCall(Context& context, SemIRLoc loc, SemIR::Call call,
     case SemIR::BuiltinFunctionKind::None:
       CARBON_FATAL() << "Not a builtin function.";
 
+    case SemIR::BuiltinFunctionKind::IntMake32: {
+      if (phase != Phase::Template) {
+        break;
+      }
+      return context.constant_values().Get(SemIR::InstId::BuiltinIntType);
+    }
+
+    case SemIR::BuiltinFunctionKind::FloatMake: {
+      if (phase != Phase::Template) {
```

Comment:
```suggestion
      // TODO: Support a symbolic constant width.
      if (phase != Phase::Template) {
```
I assume we'd eventually want to allow
```
fn F[N: i32](f: Core.Float(N))
```

Reply:
Done, although I would hope these are actually types before that happens.